index.js

```diff
@@ -70,6 +70,38 @@ Builder.prototype.build = function(moduleName, outFile, opts) {
   });
 };
 
+var operatorFunctions = {
+  '+': builder.addTrees,
+  '-': builder.subtractTrees
+};
+
+Builder.buildExpression = function(firstModule, operations, cfg) {
```

Comment:
There is the potential in future for a few other operators, like an intersection operator (maybe `^`), see https://github.com/jspm/jspm-cli/issues/133 (it's * there). There is also the possibility of weak addition and weak subtraction (adding / subtracting just a single module itself and not its full dependency tree).